src/styles/main.scss

```diff
@@ -0,0 +1,57 @@
+$width-box: 100px;
+$gap: 1px;
+$days-of-week: (mon 0, tue 1, wed 2, thu 3, fri 4, sat 5, sun 6);
+
+body {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+  margin: 0;
+  font-family: Arial, sans-serif;
+  font-size: 30px;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: 1px;
```

Comment:
You have a SCSS variable `$gap` defined at the beginning of the file. It would be better to use it here instead of hardcoding the value.